clang/lib/Driver/ToolChains/Arch/AArch64.cpp

```diff
@@ -445,4 +445,21 @@ void aarch64::getAArch64TargetFeatures(const Driver &D,
 
   if (Args.getLastArg(options::OPT_mno_bti_at_return_twice))
     Features.push_back("+no-bti-at-return-twice");
+
+  // Parse AArch64 CPU Features
+  const Arg *CPUArg = Args.getLastArg(options::OPT_mcpu_EQ);
```

Comment:
Feels like this should be done in `getAArch64ArchFeaturesFromMcpu`/`DecodeAArch64Mcpu`